### ISSUE TITLE
[IBM Qiskit Challenge][ Hybrid Algorithms Challenge][QAQA challenge][ Science Challenge][Young Scientist Challenge] Galaxy Detection by Quantum Machine Learning 

### ISSUE THREAD

**Issue by BrightSky77** [XanaduAI/QHack2022#116]
### Team Name: 

Voyager

### Project Description: 
This project is about applying qunatum machine learning in the field of astronomy. We successfully detected galaxy with accuracy of 94% by quantum machine learning model training in quantum circuit. We divided galaxy image from NASA into pixcels and used each pixcel as an input data. We coded quantum circuit 15 by using qiskit module and this circuit is from the paper [Expressibility and entangling capability of parameterized quantum circuits for hybrid quantum-classical algorithms, arXiv:1905. 10876] (https://arxiv.org/abs/1905.10876).  This circuit 15 has high expressibility. Expressibility quantifies how circuit can express freely in Hilbert Space. This circuit is used in data embedding and quantum machine learning model training.

We used L-BFGS algorithm for optimization and crossentropy for loss function in our quantum circuit model.  

### Presentation: 

Here is a power point file, video and excel file which is a result of our trained model.
**Please watch the VIDEO**

https://github.com/BrightSky77/Qhack_Quantum_Machine_Learning

### Source code: 

https://github.com/BrightSky77/Qhack_Quantum_Machine_Learning

### Which challenges/prizes would you like to submit your project for?

IBM Qiskit Challenge,  Hybrid Algorithms Challenge, QAQA challenge, Science Challenge, Young Scientist Challenge

**Comment by isaacdevlugt** [comment]
Thank you for your submission! There's still time to populate your submission with code, presentation material, etc. Please make any final adjustments before the deadline tonight at 17h00 EST!

Good luck!
